src/lib.rs

```diff
@@ -47,9 +54,24 @@ pub struct Args {
     pub vimgrep: bool,
 }
 
+impl Args {
+    pub fn use_paths(&self) -> Vec<PathBuf> {
+        if self.paths.is_empty() {
+            vec![Path::new("./").to_owned()]
+        } else {
+            self.paths.clone()
+        }
+    }
+
+    pub fn is_using_default_paths(&self) -> bool {
+        self.paths.is_empty()
+    }
+}
+
 #[derive(clap::Args)]
 #[group(required = true, multiple = false)]
 pub struct QueryArgs {
+    #[arg(short = 'Q', long = "query-file")]
```

Comment:
The path to a query file had been a positional command-line argument, I'm guessing it's uncontroversial that we should mimic grep-like tools and have paths be the positional command-line arguments?

Reply:
Yeah that seems right to me.